rticonnextdds_connector.go

```diff
@@ -71,39 +72,43 @@ type Infos struct {
 
 // Identity is the structure for identifying
 type Identity struct {
-	WriterGuid     [16]byte `json:"writer_guid"`
+	WriterGUID     [16]byte `json:"writer_guid"`
 	SequenceNumber uint     `json:"sequence_number"`
 }
 
 // SampleHandler is an User defined function type that takes in pointers of
 // Samples and Infos and will handle received samples.
 type SampleHandler func(samples *Samples, infos *Infos)
 
+const (
```

Comment:
@metarsit can you add DDSRetCodeNoData = 11?

Reply:
Added